layers/timeline_semaphore.c

```diff
@@ -1104,6 +1104,13 @@ static VkResult timeline_ImportSemaphoreFdKHR(
     return result;
 }
 
+static VkResult timeline_ImportSemaphoreFd(
```

Comment:
As far as I can tell, `vkImportSemaphoreFd` is not a Vulkan 1.1 or 1.2 symbol.

Reply:
Yes, that's why I have removed it already. Did you review commit by commit?